TabletopTweaks/Extensions/ExtentionMethods.cs

```diff
@@ -462,6 +462,28 @@ static class ExtentionMethods {
                 blueprint.AddComponent(newComponent);
             }
         }
+
+        /// <summary>
```

Comment:
This should probably by rewritten for consistency with other ReplaceComponent methods - to replace all occurences. Maybe could be added as ReplaceFirstComponent.